wiki/Article_Style_Guide/en.md

```diff
@@ -1,192 +1,278 @@
+# Article Style Guide
+
+## Introduction
+**Restructure or rewrite the whole page to GitHub-flavoured Markdown(GFM)-compliance.** HTML tags are out-of-question.
+
+GFM can be learned in one day, but do not expect too much out of it as it intended for plaintext with bare-minimum formatting needed to function as a readable file for a webpage.
+
+### Language support
+**_osu!web [as of 14/1/2017. Still WIP]_**  
+Link: <https://new.ppy.sh/wiki/>
+1. HTML tags not supported (any HTML tags and its contents will be removed from public view).
+2. Image display is literal display without proper encapsulation.
+
+**_Not supported (in GitHub view):_**
+1. HTML5 inline CSS (specifically, "style"/"class" attribute) standard,
+2. color (all types),
+3. underline (the `<u>` tag). Use **_Bold&Italic_** instead or _Italic_ depending on the paragraph styling used.
+
+## Page rule
+**_Headers:_**
+- **Level 1 header** must be the **title of the page**.
+- All other headers used must be level 2 onwards as see fit.
+- Leave a space empty before writing the header name after setting `#` (header level).
+
+**_Links:_**
+- When linking to other pages, you do not need to specify the specific language file (done by osu!web with 'locales').
+  - Capitalisation and punctuation is strict (must be spelt exactly or _404 not found_) when directing to the page.
+    - Linking to osu!catch: `/wiki/Game_Modes/osu!catch/`
+  - For directing to a specific header in the page, the header name must be in lowercase and spaces/punctuations are to be replaced by hyphen (`-`).
+    - Linking to specific section of FAQ: `/wiki/FAQ#ive-never-played-this-ouendan-thing---what-is-it/`
+- `https://` preferred.
+- When translating, keep the **english link reference name**.
+  - This is to ease maintenance in the event of image/link problem (direct copy and paste, rather than hunting for that one link to be fixed).
+
+**_Keyboard keys:_**
+- Use grave marks (\`) and [camelCase](https://en.wikipedia.org/wiki/Camel_case) format.
+  - Example 1: `;`
+  - Example 2: `D`
+  - Example 3: `Spacebar`
+  - Example 4: `leftCtrl`
+  - Example 5: `Shift`-`1`
+
+**_Page study of interest:_**
+- [FAQ](https://github.com/ppy/osu-wiki/tree/master/wiki/FAQ/en.md) {general structure, image structure style (same level)}
+- [Multiplayer](https://github.com/ppy/osu-wiki/tree/master/wiki/Multiplayer/en.md) {general structure, raw HTML table usage, image structure style (same level)}
+- [Skinning](https://github.com/ppy/osu-wiki/tree/master/wiki/Skinning/en.md) {Image structure style (img folder)}
+- [Accuracy](https://github.com/ppy/osu-wiki/tree/master/wiki/Accuracy/en.md) {Image structure style (same level)}
+- [osu!store](https://github.com/ppy/osu-wiki/tree/master/wiki/osu!store/en.md) {Extreme Markdown and HTML hybrid}
+
+## General styling
```

Comment:
Might as well make each item a subheading of themselves to include more examples (and make it look a bit more cleaner)